presto/presto.go

```diff
@@ -869,10 +869,70 @@ func (c *typeConverter) ConvertValue(v interface{}) (driver.Value, error) {
 		}
 		return v, nil
 	default:
+		if strings.HasPrefix(c.typeName, "row(") {
+			values, ok := v.([]interface{})
+			if !ok {
+				return nil, fmt.Errorf("can't convert type to row: %q", c.typeName)
+			}
+
+			rowItems := parseRowItems(c.typeName)
+			if len(rowItems) != len(values) {
+				return nil, fmt.Errorf("there should be as many types as there are values")
+			}
+
+			m := map[string]driver.Value{}
+
+			for pos, item := range rowItems {
+				converter := newTypeConverter(item.typeName)
+				value, err := converter.ConvertValue(values[pos])
+				if err != nil {
+					fmt.Printf("Error converting value of %s: %s\n", item.name, err.Error())
```

Comment:
We shouldn't use prints in production code. Should we just return from here?
```suggestion
					return nil, err
```